CONTRIBUTING.md

```diff
@@ -1,5 +1,47 @@
-# Contribute
+# Contribute to the documentation
+We really appreciate your support! Please have a look at the [issues page on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues) and check if anybody already raised an issue with the same topic.
 
-Thanks for your support!
-Please open an issue on github. More to come soon...
-~~Please open a pull request on GitHub!~~
+If this is the case, please subscribe to the issue and explain your proposal.
+
+If that's not the case, please open a [new issue](https://github.com/jamulussoftware/jamuluswebsite/issues/new/choose), select which type of issue it is and describe your issue.
+
+If you want to open a Pull Request (after having described and discussed your proposal), have a look at the [Contribution page on the website](https://jamulus.io/wiki/Contribution). Keep in mind that your changes (if they are not just typos) might have to wait until everything is translated. We will bundle them (if everything is allright) with other changes and upload multiple changes in one bulk to jamulus.io.
+
+## I don't know GitHub. What do I need to know?
```

Comment:
This could go to the website

Reply:
Yes.

Reply:
Done.

Reply:
Any thoughts on this? I've moved some files on the website but I'm not sure if that isn't too much. (also removed a few SourceForge references since I think to make the line between development and Forum clearer, most feature requests should go to github)